terra-jupyter-r/Dockerfile

```diff
@@ -1,17 +1,17 @@
-FROM us.gcr.io/broad-dsp-gcr-public/terra-jupyter-base:1.0.14
+FROM us.gcr.io/broad-dsp-gcr-public/terra-jupyter-base:1.1.0
 USER root
 
 COPY scripts $JUPYTER_HOME/scripts
 
 # Add env vars to identify binary package installation
-ENV TERRA_R_PLATFORM="terra-jupyter-r"
+ENV TERRA_R_PLATFORM="terra-jupyter-r-1.1.0"
```

Comment:
changed env var to include version to resolve issue with binary package installation, [thread here](https://the-anvil.slack.com/archives/C034WLKV15K/p1687889742101099)